src/message.rs

```diff
@@ -442,6 +436,19 @@ impl ChatMessageContent {
                 document_message.set_fileName(filename);
                 message.set_documentMessage(document_message);
             }
+            ChatMessageContent::Audio {info, dur, ptt} => {
+                let mut audio_message = message_wire::AudioMessage::new();
+                audio_message.set_url(info.url);
+                audio_message.set_mimetype(info.mime);
+                audio_message.set_fileEncSha256(info.enc_sha256);
+                audio_message.set_fileSha256(info.sha256);
+                audio_message.set_fileLength(info.size as u64);
+                audio_message.set_mediaKey(info.key);
+                audio_message.set_ptt(ptt);
+                audio_message.set_seconds(dur.as_secs() as u32);
+                //FIXME missing sidecar
```

Comment:
Just out of interest, does this not cause whatsapp to get upset (leaving out the sidecar)? (not a blocker for this PR, just curious)

Reply:
I've recorded an audio file and converted it to a single channel opus file and sent it to myself using whatsappweb-rs. I could play the file fine both on Whatsapp Web and on Whatsapp Android client.
The reverse engineering project mentioned that the sidecar is used to support random access to media file. I guess that would become more relevant for video.